packages/serverpod/lib/src/server/serverpod.dart

```diff
@@ -466,6 +466,16 @@ class Serverpod {
         securityContext: _securityContextConfig?.webServer,
       );
     }
+
+    if (Features.enableInsights) {
+      if (_isValidSecret(config.serviceSecret)) {
+        _serviceServer = _configureInsightsServer();
```

Comment:
Would it make sense to rename `_serviceServer` to `_insightServer` (or on the flip side rename the helper)?